### PR TITLE
Task Test on Dufuna

### PR DESCRIPTION
**Pull request by nifemi-omotola**
### Task: <task name here>

I <clear and concise explanation>
I <clear and concise explanation>

### CODE REVIEW

**Comment by michaelokuboyejo** [comment]
Please can you add a description for this pull request?

**Review by beccadaniel — COMMENTED** [review]
Well done @nifemi-omotola

**Comment by michaelokuboyejo** [comment]
Please can you add a description for this pull request?

**Comment by yemexx1** [comment]
👋

File: submissions/Oluwanifemi Omotola/Test for Github-Slack-Integration [hunk 1]
```diff
@@ -0,0 +1 @@
+I am testing the Github-Slack Integration
```
Comment (kenny-wealth) [inline submissions/Oluwanifemi Omotola/Test for Github-Slack-Integration:1]: Please remove spaces from file and folder names 😊

**Comment by nifemi-omotola** [comment]
This PR is no longer needed

**Pull request closed without merging**